devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java

```diff
@@ -201,6 +202,7 @@ public void execute() throws MojoExecutionException {
             QuarkusBootstrap.Builder builder = QuarkusBootstrap.builder()
                     .setAppArtifact(appArtifact)
                     .setMavenArtifactResolver(resolver)
+                    .setIsolateDeployment(true)
```

Comment:
Is there any chance of this causing unexpected behavior?

Reply:
I think the current failures are because close is being called twice, which means it attempts to define the driver remover twice

Reply:
OK, let's see what CI says